src/server.tsx

```diff
@@ -98,16 +102,25 @@ const fetchCachedEpicTests: { [key in EpicType]: () => Promise<EpicTests> } = {
     LIVEBLOG: fetchConfiguredLiveblogEpicTestsCached,
 };
 
+const getArticleEpicTests = async (): Promise<Test[]> => {
+    const configuredTests = await fetchCachedEpicTests['ARTICLE']();
```

Comment:
Doesn't really matter, but here we could do 

```js
const configuredTests = await fetchCachedEpicTests.ARTICLE();
```

Reply:
👍